src/stories/Library/event-list-item/event-list-item.scss

```
@@ -1,21 +1,64 @@
-$_event-list-item--max-width: 1126px;
-$_event-list-item--image-height: 166px;
 $_desktop-content-rows: 5;
+$_event-list-item--image-height: 166px;
+$_event-list-item--image-width: 222px;
+$_event-list-item--max-width: 1126px;
+$_stacked-reduce-width: 20px;
 
-.event-list-item {
+@mixin event-list-item-stacked-shadow-box {
+  position: relative;
+  &::before {
+    content: "";
+    background-color: $color__global-white;
+    position: absolute;
+    width: calc(100% - $_stacked-reduce-width);
+    height: 100%;
+    left: 50%;
+    top: 0;
+    transform: translateX(-50%);
+    z-index: -1;
+    box-shadow: inset 0 15px 15px -10px rgba(0, 0, 0, 0.1);
+  }
+}
+
+@mixin event-layout {
```

Nice with the mixin, i'll just add some name suggestions you can consider if you want or not. 


```suggestion
@mixin event-list-item-grid {
```